routes/badges.js

```diff
@@ -34,4 +34,37 @@ const badge = require('../controllers/badge.js')
  */
 router.get('/', badge.getBadges)
 
+/**
+ * @swagger
+ * /badges/:username:
+ *   get:
+ *     summary: Get all the badges of a particular user.
+ *
+ *     tags:
+ *       - Badges
+ *     responses:
+ *       200:
+ *         description: User badegs
+ *         content:
+ *           application/json:
+ *              schema:
+ *                type: object
+ *                properties:
+ *                  message:
+ *                    type: string
+ *                    example: User badges returned successfully!
+ *                  userBadges:
+ *                    type: array
+ *                    items:
+ *                      $ref: '#/components/schemas/userBadges'
```

Comment:
Need to add 404 for the case when username provided doesn't exist

Reply:
Yes will do that.